specification/metadata-3.2.md

```diff
@@ -111,6 +123,9 @@ for users by issuing a disabling `MONITOR` command, and for channels by
 parting the channel. Clients are automatically subscribed to notifications for
 their own metadata, excluding changes made by the clients themselves.
 
+Client subscriptions to a channel includes notifications for other users in
+the channel, regardless of `MONITOR` state.
```

Comment:
I believe the use case given for that addition is for clients that would very much want to use the `avatar`/`info`/etc set by users in their display, where otherwise they would have to resort to monitoring each of the users joined to common channels separately.

@jwheare wanna weigh in on this?